notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<p>This header seems out of place here. Maybe move this down and replace this with a header saying "Create input data pipeline with tf.data" or something along those lines. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>